src/uas_standards/astm/f3411/v22a/constants.py

```diff
@@ -25,3 +25,16 @@ class Scope(str, Enum):
 NetMinObfuscationDistanceM = 300
 NetDSSMaxSubscriptionPerArea = 10
 NetDSSMaxSubscriptionDurationHours = 24
+
+MinPositionResolution = 0.0000001
```

Comment:
Since these values are not named in the standard, it's probably worth documenting what they are and where they come from; e.g.:

```suggestion
MinPositionResolution = 0.0000001
"""Minimum resolution of both latitude and longitude values according to definitions in Table 1."""
```